documentation/source/geometry/overview.rst

```diff
@@ -0,0 +1,282 @@
+        >>> wire1 = geometry.tools.make_polygon(pntslist, label="wire1", closed=True)
+        Out: ([BluemiraWire] = Label: wire1,  length: 4.0,  area: 0.0,  volume: 0.0, )
+        >>> wire1.is_closed()
+        Out: True
+
+    Same procedure can be applied to ``make_bezier`` and ``make_bspline`` to generate
+    bezier and bspline curves, respectively.
+
+* a `BluemiraFace`:
+
+    A BluemiraFace object is defined by its boundary that must be a closed
+    BluemiraWire. If a list of BlumiraWire (with more than one wire) is given, the
+    first wire is used as outer boundary of the face, while the others are considered
+    as face holes.
+
+    .. warning:: wires representing holes must not intersect. No internal check is
+        implemented for the moment, so the check is on the user.
+
+    .. code-block:: pycon
+
+        >>> from bluemira.geometry.tools import make_polygon
+        >>> from bluemira.geometry.face import BluemiraFace
+
+        >>> pntslist_out = [(1.0, 1.0, 0.0), (0.0, 1.0, 0.0), (0.0, 0.0, 0.0), (1.0, 0.0,0.0)]
+        >>> delta = 0.25
+        >>> pntslist_in = [ (1.0 - delta, 1.0 - delta, 0.0),
+                            (0.0 + delta, 1.0 - delta, 0.0),
+                            (0.0 + delta, 0.0 + delta, 0.0),
+                            (1.0 - delta, 0.0 + delta, 0.0),
+                          ]
+        >>> wire_out = make_polygon(pntslist_out, label="wire_out",closed=True)
+        >>> bmface = BluemiraFace(wire_out)
+        Out: ([BluemiraFace] = Label: wire_out,  length: 4.0,  area: 1.0,  volume: 0.0, )
+        >>> wire_in = make_polygon(pntslist_in, label="wire_in", closed=True)
+        >>> bmface_with_hole = BluemiraFace([wire_out, wire_in],label="face_with_hole")
+        Out: ([BluemiraFace] = Label: face_with_hole,  length: 6.0,  area: 0.75,volume: 0.0, )
+
+    .. note:: the length of the face is equal to the total length of the boundary.
+
+* a `BluemiraShell`
+
+    A BluemiraShell object is defined by its boundary that must be a set of
+    BluemiraFace objects.
+
+    .. warning:: faces shall not intersect. No internal check is implemented for the
+        moment, so the check is on the user.
+
+    .. code-block:: pycon
+
+        vertexes = [(0.0, 0.0, 0.0), (1.0, 0.0, 0.0), (1.0, 1.0, 0.0), (0.0, 1.0, 0.0),
+                    (0.0, 0.0, 1.0), (1.0, 0.0, 1.0), (1.0, 1.0, 1.0), (0.0, 1.0, 1.0)]
+        # faces creation
+        faces = []
+        v_index = [(0,1,2,3),(5,4,7,6),(0,4,5,1),(1,5,6,2),(2,6,7,3),(3,7,4,0)]
+        for ind, value in enumerate(v_index):
+            wire = geo.tools.make_polygon(list(itemgetter(*value)(vertexes)),closed=True)
+            faces.append(geo.face.BluemiraFace(wire, "face"+str(ind)))
+        # shell creation
+        shell = geo.shell.BluemiraShell(faces, "shell")
+
+
+* a `BluemiraSolid`
+
+    A BluemiraSolid object is defined by its boundary that must be a closed
+    BluemiraShell object.
+
+    .. code-block:: pycon
```

Comment:
I think most of these can just be 'python' code blocks. We don't get the syntax highlighting with `pycon` as I guess it just assumes it's all console output